src/api/status/public/js/build-log/build-header.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
Why?

Reply:
It was complaining about `head_commit` not being in `camelcase`.